lighthouse-core/lib/lh-error.js

```diff
@@ -57,23 +58,14 @@ class LighthouseError extends Error {
     super(errorDefinition.code);
     this.name = 'LHError';
     this.code = errorDefinition.code;
-    this.friendlyMessage = str_(errorDefinition.message, properties);
+    const mergedProperties = {...errorDefinition.defaultProperties, ...properties};
```

Comment:
I'm not a huge fan of `defaultProperties` basically copying the `.code` over, but maybe there's a future world in which the generic is helpful. Did you have any particular use case in mind?

Reply:
yeah, this was kind of my idea, trying to tame some complexity of just trying to get the error codes through in the localized string. I'm kind of leaning in the other direction now, though, and thinking we should just have a unique UIString for each of these so we don't need the error code appended on to them?

Or is there another way we can accomplish this?